src/jimgw/cli/_likelihood.py

```
@@ -117,6 +123,68 @@ def build_likelihood(
+            if tdet_comp is not None:
+                data = ifos[0].data
+                t_end = float(data.start_time) + float(data.duration) - trigger_time
+                s = EARTH_RADIUS_LIGHT_S
+                if mb_time_offset is None:
+                    mb_time_offset = t_end - float(getattr(tdet_comp, "xmin")) + s
+                    logger.info(
+                        "time_offset inferred from t_det prior: %.4f s", mb_time_offset
+                    )
+                if mb_delta_f_end is None:
+                    mb_delta_f_end = 100.0 / (
+                        t_end - float(getattr(tdet_comp, "xmax")) - s
```

Ruff (0.15.13)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)